app/views/forms/_screen.html.erb

```diff
@@ -26,12 +26,14 @@
 
     <%= f.field(:name, :required => true, :read_only => cannot?(:rename, @form)) %>
 
+    <div class="more-settings"><a href="#"><%= t("common.more_settings") %></a></div>
     <% unless admin_mode? %>
-      <%= f.field(:smsable, :type => :check_box)  %>
-      <%= f.field(:allow_incomplete, :type => :check_box, :read_only => @form.published?) unless admin_mode? %>
-      <%= f.field(:published, :type => :check_box, :read_only => true) unless @form.new_record? %>
-      <%= f.field(:access_level, :partial => 'form_access_level') %>
-      <%= f.field(:version, :content => @form.version_with_code) unless @form.new_record? %>
+      <div class="setting_fields">
+        <%= f.field(:smsable, :type => :check_box)  %>
+        <%= f.field(:authenticate_sms, type: :check_box) %>
```

Comment:
I think this param has not been whitelisted. Please always do a full test when adding new form fields to make sure the value saves properly.